internal/nix/store_test.go

```diff
@@ -0,0 +1,32 @@
+package nix
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestContentAddressedPath(t *testing.T) {
+
+	testCases := []struct {
+		storePath string
+		expected  string
+	}{
+		{
+			"/nix/store/r2jd6ygnmirm2g803mksqqjm4y39yi6i-git-2.33.1",
+			"/nix/store/ldbhlwhh39wha58rm61bkiiwm6j7211j-git-2.33.1",
```

Comment:
Does Nix automatically download these if they're not in the store yet?

Reply:
yep